proto/api.proto

```diff
@@ -631,3 +648,19 @@ message GetTradesStreamResponse {
   int64 slot = 1;
   GetTradesResponse trades = 2;
 }
+
+message GetRecentBlockHashStreamRequest {
```

Comment:
minor - you can actually reuse the GetRecentBlockHashRequest and GetRecentBlockHashResponse messages for streaming endpoint. No need to duplicate.